lib/SPIRV/SPIRVLowerMemmove.cpp

```diff
@@ -54,54 +54,51 @@ namespace SPIRV {
 
 class SPIRVLowerMemmoveBase {
 public:
-  SPIRVLowerMemmoveBase() : Context(nullptr), Mod(nullptr) {}
+  SPIRVLowerMemmoveBase() : Context(nullptr) {}
+
   void LowerMemMoveInst(MemMoveInst &I) {
+    // There is no direct equivalent of @llvm.memmove in SPIR-V and the closest
+    // instructions are 'OpMemoryCopy' and 'OpMemoryCopySized'.
```

Comment:
```suggestion
    // instructions are 'OpCopyMemory' and 'OpCopyMemorySized'.
```

same applicable below

Reply:
Done in dbc54dd752c04bb65e8d2e2aedaaac22c247266e